### PR TITLE
v1.1.0_SSR-1375

### PR DESCRIPTION
**Pull request by nmanhit**

### CODE REVIEW

File: src/mobile/dropdown/index.ts [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+        >
+      </label>
+      <div class="kuc-mobile-dropdown__input-form">
+        <div class="kuc-mobile-dropdown__input-form__select">
+          <select
+            class="kuc-mobile-dropdown__input-form__select__input"
+            id="${this._GUID}-label"
+            aria-describedBy="${this._GUID}-error"
+            aria-required="true"
+            aria-invalid="true"
```
Comment (josh-vo) [inline src/mobile/dropdown/index.ts:131]: The "aria-required" and "aria-invalid" have condition to set "true". So, could you please update code following spec?

**Review by josh-vo — CHANGES_REQUESTED** [review]

**Review by josh-vo — APPROVED** [review]

File: src/mobile/dropdown/test/className.test.ts [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import { expect, fixture } from "@open-wc/testing";
+import { MobileDropdown } from "../index";
+
+describe("MobileDropdown", () => {
+  describe("className", () => {
+    it("className default prop is null", async () => {
```
Comment (trung-doan) [inline src/mobile/dropdown/test/className.test.ts:6]: @nmanhit 
Please remove `className` and change the `title` to the same content with expectation.
Ex: `it("should be set to empty string when initializing without props option")`

File: src/mobile/dropdown/test/className.test.ts [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+      const container = new MobileDropdown();
+      const el = await fixture(container);
+      expect(el.classList.length).to.be.equal(0);
+    });
+
+    it("className default prop set successfully", async () => {
```
Comment (trung-doan) [inline src/mobile/dropdown/test/className.test.ts:12]: @nmanhit 
Please remove `className` and change the `title` to the same content with expectation.
Ex: `it("should be equal \"options-class\" string when initializing className with \"options-class\" value ")`

File: src/mobile/dropdown/test/className.test.ts [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+      const container = new MobileDropdown({ className: "options-class" });
+      const el = await fixture(container);
+      expect(el.classList.length).to.be.equal(1);
+      expect(el.className).to.have.equal("options-class");
+    });
+
+    it("className prop replace successfully'", async () => {
```
Comment (trung-doan) [inline src/mobile/dropdown/test/className.test.ts:19]: @nmanhit 
Please remove `className` and change the `title` to the same content with expectation.
Ex: `it("it should be replace by \"replace-class\" when changing by setter")`

File: src/mobile/dropdown/test/className.test.ts [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+      const container = new MobileDropdown({
+        className: "options-class"
+      });
+      container.className = "replace-class";
+      const el = await fixture(container);
+      expect(el.classList.length).to.be.equal(1);
+      expect(el.className).to.have.equal("replace-class");
+    });
+
+    it("className default prop set to null", async () => {
```
Comment (trung-doan) [inline src/mobile/dropdown/test/className.test.ts:29]: @nmanhit 
Please remove this unneeded case.

File: src/mobile/dropdown/test/className.test.ts [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+      // @ts-ignore
+      const container = new MobileDropdown({ className: null });
+      const el = await fixture(container);
+      expect(el.classList.length).to.be.equal(1);
+      expect(el.className).to.have.equal("null");
+    });
+
+    it("className prop set to null", async () => {
```
Comment (trung-doan) [inline src/mobile/dropdown/test/className.test.ts:37]: @nmanhit 
Please remove this unneeded case.

**Review by trung-doan — CHANGES_REQUESTED** [review]

**Pull request closed without merging**